src/greyVest/utils.js

```diff
@@ -1,6 +1,7 @@
 import F from 'futil-js'
 
-export let openBinding = (...lens) => ({
-  isOpen: F.view(...lens),
-  onClose: F.off(...lens),
-})
+export let openBinding = lens =>
```

Comment:
We still need the spread here to support all the futil lens types, so with no args `lens` would be `[undefined]` instead of `undefined`. Maybe something like

```js
export let openBinding = (...lens) => 
  !_.isEmpty(lens) && _.head(lens) && {
    isOpen: F.view(...lens),
    onClose: F.off(...lens),
  }
```